### PR TITLE
fix: update the aggregate-v2 processor node to match API changes

### PR DESCRIPTION
**Pull request by dhable**
Significant changes were required in the aggregate-v2 API schema to deliver a better UX in the web application. This commit updates the aggregate-v2 provider to match those schema changes.

Ref: LOG-19710

### CODE REVIEW

File: internal/provider/models/processors/aggregate_v2.go [hunk 1]
```diff
@@ -85,74 +78,47 @@ var AggregateV2ProcessorResourceSchema = schema.Schema{
...
+	userConfig["window"] = windowConfig
 }
 
-func strategyConfigFromModel(plan *AggregateV2ProcessorModel, userConfig map[string]any, dd *diag.Diagnostics) {
-	strategySet := !(plan.Strategy.IsNull() || plan.Strategy.IsUnknown())
-	scripSet := !(plan.Script.IsNull() || plan.Script.IsNull())
-	if !strategySet && !scripSet {
+func evaluateConfigFromModel(plan *AggregateV2ProcessorModel, userConfig map[string]any, dd *diag.Diagnostics) {
+	isOperationSet := !(plan.Operation.IsNull() || plan.Operation.IsUnknown())
+	isScripSet := !(plan.Script.IsNull() || plan.Script.IsNull())
```
Comment (mdeltito) [inline internal/provider/models/processors/aggregate_v2.go:102]: should one of these be `plan.Script.IsUnknown()` ? (I realize you just moved/renamed this declaration)

File: internal/provider/models/processors/test/aggregate_v2_test.go [hunk 2]
```diff
@@ -223,127 +223,99 @@ func TestAggregateV2Processor(t *testing.T) {
...
-						"mezmo_pipeline.test_parent2",
-						"mezmo_aggregate_v2_processor.test_processor",
-					),
-				),
-				// verify resource will be re-created after refresh
-				ExpectNonEmptyPlan: true,
-			},
-
-			// Delete testing automatically occurs in TestCase
+			//	// Error: server-side validation for missing strategy
```
Comment (mdeltito) [inline internal/provider/models/processors/test/aggregate_v2_test.go:238]: can this be uncommented or removed?

File: internal/provider/models/processors/aggregate_v2.go (hunk 1 above)
Comment (dhable) [reply internal/provider/models/processors/aggregate_v2.go:102]: Good catch. I'll fix it.

File: internal/provider/models/processors/test/aggregate_v2_test.go (hunk 2 above)
Comment (dhable) [reply internal/provider/models/processors/test/aggregate_v2_test.go]: Yup, I'll do that. These are now being validated by the schema change so these test scenarios can no longer make it to our code.

**Review by mdeltito — APPROVED** [review]
nice, LGTM

**Pull request merged**